src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowCreate.cs

```diff
@@ -152,35 +123,7 @@ public override EditRow GetEditRow(DbCellValue[] cachedRow)
         /// <returns>INSERT INTO statement</returns>
         public override string GetScript()
         {
-            // Process all the cells, and generate the values
-            List<string> values = new List<string>();
-            for (int i = 0; i < AssociatedResultSet.Columns.Length; i++)
-            {
-                DbColumnWrapper column = AssociatedResultSet.Columns[i];
-                CellUpdate cell = newCells[i];
-
-                // Skip columns that cannot be updated
-                if (!column.IsUpdatable)
-                {
-                    continue;
-                }
-
-                // If we're missing a cell, then we cannot continue
-                if (cell == null)
-                {
-                    throw new InvalidOperationException(SR.EditDataCreateScriptMissingValue);
-                }
-
-                // Format the value and add it to the list
-                values.Add(SqlScriptFormatter.FormatValue(cell.Value, column));
-            }
-            string joinedValues = string.Join(", ", values);
-
-            // Get the start clause
-            string start = GetTableClause();
-
-            // Put the whole #! together
-            return string.Format(InsertCompleteScript, start, joinedValues);
+            return BuildInsertScript(false).ScriptText;
```

Comment:
Again, what does the `false` do? Param name like `includeXyz: false` would be great.